src/PowerSystems.jl

```diff
@@ -210,13 +210,15 @@ export get_dynamic_components
 export solve_powerflow!
 export solve_powerflow
 export validate_connectivity
+export find_connected_components
```

Comment:
Is there any reason to export this?

Reply:
I think it's useful because it returns the buses that makeup the different components. I created it because I had developed a workflow where I parsed a dataset that had multiple connected components and I needed to figure out what they were to address the issue. If I'm the only one, I can not export, but I think others may find it useful.